### DISCUSSION TITLE
The Error keeps occur in final step while Installing the Hac MIni Patches (HaCMini.pkg)

### DISCUSSION THREAD

**Discussion by PBNJane** [osy86/HaC-Mini#994] [Q&A]
1. I'm using VMware with macOS 10.14 on Windows 10 Pro, because I don't have any Real Mac in my properties. So entire process was on VMware Machine.

2. I successfully install macOS 10.15.7.02 (Catalina) on My USB Memory. so that if I reboot the VMware Machine with the 'Installation USB' inserted, It starts with macOS Installation Process. And I also tried next steps on other devices that verified as proper media for Windows 10 or Linux Installation, but the Symptoms are the same.

3. I Downloaded HacMini Installer 3.1.5 as 'HacMini.pkg' File, and I successfully execution the Installer. I followed every steps on this link : https://osy.gitbook.io/hac-mini-guide/installation-guide/installation, but It all stuck in the last step without any explanation

![Screen Shot 2022-03-30 at 5 55 01 PM](https://user-images.githubusercontent.com/102657055/160793411-a59fc636-13e4-4286-8a59-1b95d9888757.png)
![Screen Shot 2022-03-30 at 6 10 56 PM](https://user-images.githubusercontent.com/102657055/160796731-43f9792c-a4e2-4b54-aeb8-ff5edccdcaf9.png)
![Screen Shot 2022-03-30 at 5 55 28 PM](https://user-images.githubusercontent.com/102657055/160793407-784b3b63-3f51-493c-bbae-59871bfe57cb.png)

4. I can't find the reason of this despites of three hours of Searching and Modifying the Settings or else.. So I need a help of this...

Besides, I'm begginner of Mac, Hac, and github.. so I'll amazingly appreciate your helps and guides of this matters.

**Comment by osy** [comment]
Do not check patch installer. Install macos on your nuc and then run the installer FROM your newly installed macos.
